packages/schema-extract/src/file-linker.ts

```diff
@@ -47,7 +48,10 @@ export class SchemaLinker {
                 refEntity = importSchema.definitions![entityType];
             }
         }
-        return refEntity ? refEntity : null;
+        if (refEntity && isRef(refEntity)) {
```

Comment:
Not TDD. I need to add a test for this but I am not sure how to recreate this issue